Makefile

```diff
@@ -0,0 +1,53 @@
+#
+
+.PHONY: clean
+clean: ## clean project
+clean:
```

Comment:
There are several cases where we have two label like here, is that correct? 
```
clean:  ## clean project
clean: 
```

Reply:
That is by design, it is for the `help` target to parse the description of each make target so taht it displays when the user runs `make help`

Reply:
Thanks